packages/common/actions/currentUser.js

```diff
@@ -125,6 +127,13 @@ export const setDefaultListId = listId => ({
   },
 });
 
+export const addTeamToTeamsList = teamId => ({
```

Comment:
Let's rename this function to addTeamToCurrentUserTeamList to clarify in which list, 'cause addTeamToTeamsList is reading by me like add to entities -> team -> byId.

Reply:
Renamed